test/contract/generatepin/generatePin.js

```diff
@@ -0,0 +1,103 @@
+const expect = require('chai').expect;
+const config = require('app/config');
+const FormatUrl = require('app/utils/FormatUrl');
+const request = require('supertest');
+
+const VALIDATION_SERVICE_URL = config.services.validation.url;
+
+/* eslint no-console: 0 no-unused-vars: 0 */
+describe.only('Pin Creation API Tests', function() {
+
+    const pinServiceUrl = FormatUrl.format(VALIDATION_SERVICE_URL, '/pin');
+
+    describe('1. Invalid number which should produce a 400 Bad Request', function () {
+        it('Returns HTTP 400 status', function (done) {
```

Comment:
Can you change all function declarations in this file to use arrow functions?

Reply:
Done for all the generate pin tests and address lookup tests.